src/views/offering/AddDiskOffering.vue

```diff
@@ -358,6 +358,9 @@ export default {
       }
     },
     isAdmin () {
+      if (!['Admin'].includes(this.$store.getters.userInfo.roletype)) {
+        this.isPublic = false
+      }
```

Comment:
Same as above.
Instead setting isPublic to false here. `handleSubmit` should be checked for the value and visibility of `a-select` element for domain must be fixed for domain admin

Reply:
@ravening correct way, in my opinion, would be adding following line in `mounted()` method of `AddComputeOffering.vue` and `AddDiskOffering.vue`,
`
this.isPublic = this.isAdmin()`

Reply:
Yes this is fine. I will make changes for this part